### DISCUSSION TITLE
Run the command after refreshing the page

### DISCUSSION THREAD

**Discussion by devEcommercePL** [filamentphp/filament#11149] [Help]
### Package

Panel builder

### Package Version

v3.2.17

### How can we help you?

What is the best way to execute the command or job
`example: Artisan::call(Run HealthCheck Command::class);`

after every Page load (example **\health-check-results**)?

**Comment by leandrocfe** [comment]
Maybe a middleware?
https://laravel.com/docs/10.x/middleware

You can use the middleware in the panelProvider

```php
$panel
...
->middleware([
...
MyMiddleware::class
])
...
```

**Reply by devEcommercePL** [reply]
I read about this feature
https://filamentphp.com/docs/3.x/panels/configuration#applying-middleware

but I thought that maybe there is another function (on Page/BasePage class) or Event/Listeners hat I can use
